app/controllers/tasks_controller.rb

```diff
@@ -39,6 +39,18 @@ def destroy
     redirect_to tasks_url, notice: "Task was successfully destroyed."
   end
 
+  def complete
```

Comment:
There isn't anything wrong with this approach but two ideas:

1) It's best practice to switch with regular CRUD methods (index, edit, etc...).  So, if possible, it would be nice to create a separate controller instead of tacking on this method here.

2)  Since we'll probably need some sort of log of when/how many tasks are completed, what do you think about setting up a new model (`TaskActions`, or similar)?